### DISCUSSION TITLE
How to correct pricing overview?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#5556] [Q&A] [answered]
I've run into a issue where during a few recent purchase orders the pricing of the components entered was incorrect.  This has therefore effected the price tracking of the component and the build.

Not 100% how to correct this. I've removed all the stock from one of the items and re-added at the correct price, but the  (incorrect) maximum overall price still show the pricing from the incorrect value entered.

How do I change or reset this? The price is linked to the purchase order, which I can't seem to remove. 

![Screenshot from 2023-09-16 17-42-46](https://github.com/inventree/InvenTree/assets/1429043/b9c7569d-670e-4115-bc8e-767a1b6802f8)

![Screenshot from 2023-09-16 17-43-56](https://github.com/inventree/InvenTree/assets/1429043/e4fb7d4d-f308-460d-8857-b26b17242c4f)

**Comment by SchrodingersGat** [comment]
Can you adjust the incorrect price on the purchase order?

**Reply by StormTrooper** [reply]
The purchase order is completed so I don't see any way to adjust it.

**Reply by SchrodingersGat** [reply, marked as answer]
You should be able to edit it from the admin interface in a pinch

**Reply by StormTrooper** [reply]
Thanks, that did it. I didn't know you could edit things like that in the admin interface. Very useful.
